packages/pangea-sdk/PangeaCyber.Net/Redact/Requests/RedactStructuredRequest.cs

```diff
@@ -40,6 +40,16 @@ public class RedactStructuredRequest : BaseRequest
         [JsonProperty("redaction_method_overrides")]
         public RedactionMethodOverrides? RedactionMethodOverrides { get; private set; }
 
+        /// <summary>
+        /// Is this redact call going to be used in an LLM request?
+        /// </summary>
+        [JsonProperty("llm_request")]
+        public bool? LLMrequest { get; private set; }
```

Comment:
Consider `LlmRequest` or `LLMRequest` instead. That's closer to how we've named other things like  `RedactionMethodOverrides.FpeAlphabet` or `FPEAlphabet`.

Reply:
Fixed